### ISSUE TITLE
Overlap of content on zoomed in map pages

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#1194]
**Please complete the following information** 
Site repository URL: All
Open SDG version: All

**Describe the bug**
When the page is zoomed up to 400%, content overlaps so that it is not possible to read. 

**To Reproduce**
1. Go to https://sdgdata.gov.uk/3-a-1/
2. Zoom into 400%
3. Click on Map tab
4. Zoom in
5. Scroll down to bottom of map 
5. See error

**Expected behavior**
No overlap of content when zoomed in

**Screenshots**
![image](https://user-images.githubusercontent.com/43404358/117295100-55d9c180-ae6b-11eb-8349-f3724c0fbd7f.png)

**Additional context**
From accessibility report (NOTE they also mentioned making sure there is no two direction scrolling but maps are exempt from this)
Solution: 
Ensure that when users increase the size of pages to 400% and the browser width 1280px to view the page content in one column, they can do this without having to scroll in two directions, and without the loss of content or functionality.For more information, please refer to [F102: Failure of Success Criterion 1.4.10 due to content disappearing and not being available when content has reflowed](https://www.w3.org/WAI/WCAG21/Techniques/failures/F102).

Please note that the reflow guidelines specify that maps are considered exceptional content as they may often require two-dimensional layout to preserve their usage and meaning. However, the associated user interface controls should not overlap.

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin My first thought was to try and make these elements (year slider, distance legend, and attribution) stack vertically on small-screens/high-zooms. But after a few hours investigation that is looking to be either impossible or a very high level of effort, so I wanted to bring that to your attention. Here are a couple of alternative ideas for quick fixes:

1. Hide the attribution on small-screens/high-zooms (similar to how we are already hiding that top-right "selection legend"). I think this would resolve the most common case on 400% zoom, which is that the attribution overlaps with the year-controls.
2. Set an arbitrary number of pixels as the maximum width of the attribution on small-screens/high-zooms. This may accomplish the same result as above but would keep the attribution visible.

By the way, I've also noticed an issue where the map is extremely short if the page is loaded while at 400% zoom. This wasn't mentioned in the ticket but seems related. I can add a quick-fix for this by instituting an arbitrary number of pixels as the minimum height of the map (like maybe 400px).

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin I just put up a PR that includes the second approach mentioned above, plus that height fix. Here is what it looks like in Firefox at 400% zoom:

![leaflet-zoom-max-width-attribution](https://user-images.githubusercontent.com/1319083/121355715-28ff4b00-c8fe-11eb-9a2c-b78795f71896.png)
